File: app/src/main/java/com/example/maksim/testapp/list/model/repository/LocalListRepositoryListener.java
package com.example.maksim.testapp.list.model.repository;

import com.example.maksim.testapp.list.model.data.GitHubUser;

import java.util.List;

/**
 * Created by Maksim on 2017-09-10.
 */

public interface LocalListRepositoryListener {
    void onUsersLoaded(List<GitHubUser> users);
}

File: app/src/main/java/com/example/maksim/testapp/list/model/repository/LocalListRepositoryListener.kt
package com.example.maksim.testapp.list.model.repository

import com.example.maksim.testapp.list.model.data.GitHubUser

/**
 * Created by Maksim on 2017-09-10.
 */

interface LocalListRepositoryListener {
    fun onUsersLoaded(users: MutableList<GitHubUser>?)
}
